Objects/longobject.c

```diff
@@ -6773,3 +6774,102 @@ int PyLong_AsUInt64(PyObject *obj, uint64_t *value)
 {
     LONG_TO_UINT(obj, value, "C uint64_t");
 }
+
+
+const PyLongLayout PyLong_LAYOUT = {
+    .bits_per_digit = PyLong_SHIFT,
+    .digits_order = PY_LITTLE_ENDIAN ? -1 : 1,
+    .endian = -1,  // least significant first
```

Comment:
```suggestion
    .endian = PY_LITTLE_ENDIAN ? -1 : 1,
    .digits_order = -1,  // least significant first
```
I think this should be reversed:)